README.md

```diff
@@ -57,121 +64,73 @@ This setup includes:
 
-```sh
-docker compose -f docker-compose/compose.only-browser.yml up
-```
+To check what other options exist to run the Mnestix Browser, see the yarn scripts in `package.json`. Highlights are:
+ - `yarn dev` to start the browser in a hot reloading dev environment.
+ - `yarn prettier`, `yarn format` and `yarn lint` to apply code formatting and linting.
+ - `yarn test` and `yarn test:headless` to run cypress tests.
+ - `yarn docker:prod` will build everything with the production flag.
+ - `yarn docker:prune` will stop all docker containers, remove them from the list and prune all volumes. Start with a blank slate :)
 
-E2E Testing with Cypress:
+## Docker Compose files
 
-```sh
-yarn docker:test
-```
+-   **compose.yml** - runs Mnestix Browser in production environment. Production image will be build if not found in local Docker Image Cache.<br>
+    **Mnestix Browser on port 3000 - http://localhost:3000**
+    <br><br>
 
-or
+-   **docker-compose/compose.dev.yml** - override file to run Mnestix Browser in a development environment. A development image will be built if it is not found in the local Docker Image Cache.<br>
+    **Mnestix Browser on port 3000 - http://localhost:3000** <br>
+    **Mnestix Api on port 5064 - http://localhost:5064** <br>
+    **AAS Repo on port 8081 - http://localhost:8081/swagger-ui/index.html** <br><br>
 
-```sh
-docker compose -f docker-compose/compose.test.yml up
-```
+-   **docker-compose/compose.test.yml** - override file used to configure and run end-to-end (E2E) tests using Cypress. When this file is executed, it will start the necessary services for the application and execute the Cypress tests.
+    If any test fails, the results and logs will be saved in a designated directory for further analysis.
 
--   #### Stopping containers:
-    Specify the correct file to stop services:
+The files in the `docker-compose` directory are [override compose files](https://docs.docker.com/compose/multiple-compose-files/merge/), which must be added with the `-f <filename>` flag (Look inside the `package.json` for examples).<br>
+The services are grouped into three [compose profiles](https://docs.docker.com/compose/profiles/): `basyx`, `backend` and `frontend`.
+They can be started together without defining `--profile` or separately by adding `--profile <profilename>` to the docker command.
 
-```sh
-docker compose -f docker-compose/compose.*.yml down
-```
+Additional services used by the Mnestix browser:
+ - **mnestix-api** - API service from the Mnestix ecosystem designed to expand Mnestix Browser functionalities, adding AAS List, Template Builder and allowing for the configuration of custom settings such as themes and aasId generation. (**On port 5054 - http://localhost:5064/swagger/index.html#/**)
+ - **mongodb** - NoSql database to store data
+ - **aas** - service of AAS repository (BaSyx component [aas-environment](https://github.com/eclipse-basyx/basyx-java-server-sdk/tree/main/basyx.aasenvironment))
```

Comment:
what about registry, submodel registry and discovery service?

Reply:
they are started through the root compose file
`docker compose -f compose.yml -f docker-compose/compose.test.yml up`

Reply:
thx for adding